spec/suma/api/ledgers_spec.rb

```diff
@@ -7,31 +7,32 @@
   include Rack::Test::Methods
 
   let(:app) { described_class.build_app }
-  let(:member) { Suma::Fixtures.member.create }
+  let(:member) { Suma::Fixtures.member.with_cash_ledger.create }
```

Comment:
What if there is no cash ledger? You added a precondition to how your code functions (cash ledger must exist) but don't test what happens if that condition is not met.